drivers/display/display_st7735r.c

```diff
@@ -23,15 +24,14 @@
 #include <zephyr/logging/log.h>
 LOG_MODULE_REGISTER(display_st7735r, CONFIG_DISPLAY_LOG_LEVEL);
 
-#define ST7735R_RESET_TIME              K_MSEC(1)
```

Comment:
Do we need to change this?

Reply:
Yes, we do- the MIPI DBI reset API takes a delay time in milliseconds, not a k_timeout_t structure. Therefore we need to pass an integer reset delay here, not `K_MSEC(1)`